docs/content/scripts/analytics/google-analytics.md

```diff
@@ -67,6 +68,7 @@ export default defineNuxtConfig({
       scripts: {
         googleAnalytics: {
           id: '', // NUXT_PUBLIC_SCRIPTS_GOOGLE_ANALYTICS_ID
+          dataLayerName: 'defaultGa' // NUXT_PUBLIC_SCRIPTS_GOOGLE_ANALYTICS_DATALAYERNAME
```

Comment:
i think this gets title cased `NUXT_PUBLIC_SCRIPTS_GOOGLE_ANALYTICS_DATA_LAYER_NAME (but I don't think we need it here)